pom.xml

```diff
@@ -40,6 +40,16 @@
             <artifactId>log4j-core</artifactId>
             <version>2.17.1</version>
         </dependency>
+        <dependency>
```

Comment:
I don't think we need to include core library since it is included as a dependency to jackson-databind automatically.

https://github.com/FasterXML/jackson-databind#maven

Reply:
I'll look into it